### PR TITLE
Add relative path for script

### PR DESCRIPTION
**Pull request by KrekoG**
## Features
- Add relative path for after.sh script

[Delivers: [#143108995](https://www.pivotaltracker.com/story/show/143108995)]

### CODE REVIEW

**Comment by KrekoG** [comment]
@srleach Ready for your review.

File: after.sh [hunk 1]
```diff
@@ -4,7 +4,9 @@ echo "============================================"
 echo " Homestead - PROVISIONING"
 echo "============================================"
 
-cd /var/www/basket/
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (srleach) [inline after.sh]: Has this been tested?

Comment (KrekoG) [inline after.sh]: Yes, with echo.

Comment (srleach) [inline after.sh]: Yes, but have you tested the script still works where it's used?

Comment (KrekoG) [inline after.sh]: Nope, on it

**Comment by KrekoG** [comment]
I should have tested this before as it does not work as I believed it does (vagrant copies the script to the /tmp folder before running it), but due to this is the provisioning script for the dev environment, it is not supposed to be the goal for this task anyway. Closing pull request.

**Pull request closed without merging**